rosdep/python.yaml

```diff
@@ -5341,6 +5369,18 @@ python3-bokeh-pip:
   ubuntu:
     pip:
       packages: [bokeh]
+python3-bosdyn-api:
+  debian: [python3-bosdyn-api]
+  ubuntu: [python3-bosdyn-api]
```

Comment:
This key says there is a debian package named `python3-bosdyn-mission`, but I wasn't able to find one. If these are only available on PyPI then I think what @tfoote is asking for is an end result like:

```yaml
python3-bosdyn-api-pip:
  debian:
    pip:
      packages: [bosdyn-api]
  ubuntu:
    pip:
      packages: [bosdyn-api]
```

That is, please replace the `python3-bosdyn-*` keys and `bosdyn-*` keys with `python3-bosdyn-*-pip` keys using the content from the `bosdyn-*` keys.